### ISSUE TITLE
[false-positive] UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD reported when field used

### ISSUE THREAD

**Issue by Regyl** [spotbugs/spotbugs#2271]
To simplify, I have these two classes
```java
public abstract class A {
protected static field String value;
}

public class B extends A {
public static String getValue() {
return value;
}
}
```

And in this case spotbugs (v. 4.2.2) shows bug with code UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD.

**Comment by gtoison** [comment]
I tried to reproduce the problem but the bug is not detected with that sample, so maybe something else was going, or the issue was fixed in the meantime
